config/Mekanism/tiers.toml

```diff
@@ -29,28 +29,28 @@
 	[tier.fluid_tanks]
 		#Storage size of Basic fluid tanks in mB.
 		#Range: > 1
-		basicStorage = 32000
+		basicStorage = 5000
```

Comment:
Why are fluid tanks being adjusted? We redid their values in 10.1.0 and I am relatively happy with their updated values.